mllib/src/test/scala/org/apache/spark/ml/optim/aggregator/DifferentiableLossAggregatorSuite.scala

```diff
@@ -157,4 +160,38 @@ object DifferentiableLossAggregatorSuite {
       this
     }
   }
+
+  /** Get feature and label summarizers for provided data. */
+  private[ml] def getRegressionSummarizers(
```

Comment:
Seems like these convenience functions could be adapted  to also be used in both in `train` logic across the linear models as well as here in the tests? Since the `seqOp` and `comboOp` are defined multiple times everywhere.

Though that could be done in a later clean up PR perhaps?